custom/icds_reports/templates/icds_reports/icds_app/poshan-progress-dashboard.directive.html

```diff
@@ -29,7 +29,9 @@ <h3 class="black">No Data</h3>
                 <tr ng-repeat="indicators in section.indicators">
                   <td ng-repeat="indicator in indicators" class="d-flex justify-content-between col-xs-6 px-0">
                     <p ng-bind="indicator.name" class="col-xs-8 no-margin"></p>
-                    <p ng-bind="indicator.value" class="col-xs-4 text-center no-margin"></p>
+                    <p class="col-xs-4 text-center no-margin">
+                      {$ (indicator.value == 0 || indicator.value == '0.00%' || !indicator.value) ? 'Not Launched' : indicator.value $}
```

Comment:
hey, we should not be doing this like this here. Because it could be possible that the location is launched but got 0% for a particular indicator. Then it would show now launched there.
Otherwise, We should check for the "num awc covered" indicators which is returned by the Ajax call. If that is 0 march all as unlaunched.

Reply:
But I'm not getting anything else in the network response. Just indicators and its respective values
Tagging @ShubhamBansal1997 for more inputs

Reply:
Do you mean that you get the indicator name and their value right?. So you must also be getting the indicator name "Number of AWC launched", That is the indicator which should be used to judge whether the location selected is launched or not. Is the value of the indicator is >=1 then the location is launched otherwise unlaunche.d